src/validation-renderer.js

```diff
@@ -2,19 +2,18 @@ import {DOM} from 'aurelia-pal';
 
 export class ValidationRenderer {
   renderErrors(node, relevantErrors) {
+    this.unrenderErrors(node);
     if (relevantErrors.length) {
       node.parentElement.classList.add('has-error');
       relevantErrors.forEach(error => {
         if (node.parentElement.textContent.indexOf(error.message) === -1) {
           let errorMessageHelper = DOM.createElement('span');
           let errorMessageNode = DOM.createTextNode(error.message);
           errorMessageHelper.appendChild(errorMessageNode);
-          errorMessageHelper.classList.add('help-block');
+          errorMessageHelper.classList.add('help-block', 'au-validation');
```

Comment:
I think this change is also in your other PR as well can you back the changes out for this one and I can rebase / merge them back on top of master while merging?  You can do something like this if you aren't too familiar with GitHub - 

```
$ git reset HEAD~1
$ git checkout -- src/validation-renderer.js
$ git add .
$ git commit
```